.travis.yml

```diff
@@ -1,12 +1,24 @@
+env:
+  global:
+    # TODO: We need to add the reporter id here
+    # SEE: https://docs.codeclimate.com/docs/finding-your-test-coverage-token#section-how-to
+    - CC_TEST_REPORTER_ID=<some-id-here>
+
 language: ruby
 cache: bundler
 
 rvm:
-  - 2.3.1
-  - 2.2.5
+  - 2.5.0
 
 before_script:
   - cp spec/dummy/config/database.yml.travis spec/dummy/config/database.yml
+  # Set up cc test reporter
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+  - chmod +x ./cc-test-reporter
+  - ./cc-test-reporter before-build
```

Comment:
How about just remove the Code climate thing?

Reply:
I agree. Especially if the plan is to remove ninetails anyway, there's no point spending time making codeclimate work just for a badge on the readme! haha